chips/cc2650/src/chip.rs

```diff
@@ -0,0 +1,43 @@
+use cortexm3::nvic;
+use gpio;
+use kernel;
+use peripheral_interrupts::*;
+
+pub struct Cc2650(());
+
+impl Cc2650 {
+    pub unsafe fn new() -> Cc2650 {
+        Cc2650(())
+    }
+}
+
+impl kernel::Chip for Cc2650 {
+    type MPU = ();
+    type SysTick = ();
+
+    fn mpu(&self) -> &Self::MPU {
+        &self.0
+    }
+
+    fn systick(&self) -> &Self::SysTick {
+        &self.0
+    }
+
```

Comment:
This implies that SysTick and MPU is not enabled yet perhaps leave a comment somewhere at it.